runtime/moonriver/src/lib.rs

```diff
@@ -1355,8 +1432,13 @@ where
 {
 	fn convert(currency: CurrencyId) -> Option<MultiLocation> {
 		match currency {
+			// For now (and until we upgrade to 0.9.16 is adapted) we need to use the old anchoring
+			// here
+			// This is not a problem in either cases, since the view of the destination chain
+			// does not change
+			// TODO! change this to NewAnchoringSelfReserve once we uprade to 0.9.16
```

Comment:
We have already upgraded to  0.9.16

Reply:
the comment is actually wrong. Its Xtokens that needs to be adapted to this case. I will update the comment to reflect that